account-kit/rn-signer/example/src/screens/otp-auth.tsx

```diff
@@ -11,23 +11,24 @@ import {
 
 import Config from "react-native-config";
 import { RNAlchemySigner } from "@account-kit/react-native-signer";
-export default function MagicLinkAuthScreen() {
+
+const signer = RNAlchemySigner({
+  client: { connection: { apiKey: Config.API_KEY! } },
+});
```

Comment:
shouldn't this still work within the a specific screen since it always returns a singleton now?

Reply:
Yeah, did this to silence a linting warning we were getting since the signer instance is also used in a `useEffect`